library/alloc/src/collections/btree/map.rs

```diff
@@ -421,6 +449,23 @@ impl<K: Debug, V: Debug, A: Allocator + Clone> Debug for IntoIter<K, V, A> {
     }
 }
 
+#[stable(feature = "default_iters", since = "CURRENT_RUSTC_VERSION")]
+impl<K, V, A> Default for IntoIter<K, V, A>
+where
+    A: Allocator + Default + Clone,
```

Comment:
Hmm, I would have guessed just
```suggestion
    A: Default,
```
but it's unstable and we can weaken it later, so meh.